data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
What we should have is a single browser definition place, like in
"esnext-browsers.js", and remove all other form inside the data files.  I
an willing to do that in a separate patch, but only after this pull queue
is emptied to avoid all the merge issues.

On Fri, Aug 26, 2016 at 4:07 PM, Sergey Rubanov <notifications@github.com>
wrote:

> In data-non-standard.js
> <https://github.com/kangax/compat-table/pull/891#discussion_r76471565>:
>
> > @@ -1218,7 +1212,7 @@ exports.tests = [
> >      firefox12: false,
> >      firefox16: true,
> >      firefox17: false,
> > -    firefox20: true,
> > +    firefox20: false,
> >      firefox28: false,
>
> But for non-standard features table, which is the one we are talking about
> here (data-non-standard.js), no browsers are marked as "very obsolete".
>
> Wow. Didn't notice. They should match obsolete browsers for other tables
> IMHO. Did someone had any counterarguments?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kangax/compat-table/pull/891/files/a35c2e4dd8c74a8732752e197bb497c094aa209f#r76471565>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABRIeBTuTd-RdIVGDrBG9ICby5eeEwlMks5qjzlpgaJpZM4JsXuy>
> .
>